state_machine/unit_tests/test_igrf.py

```diff
@@ -31,28 +28,28 @@ def test_matches_IGRF13(self):
         # using pyIGRF.py
```

Comment:
Can't comment there for some reason but line 21 is another duplicate use of EARTH_RADIUS. This should be referenced from one single place in all of the code.

Reply:
I rather not do that for test cases, I want the inputs to the functions to be fixed regardless of if we update earth's radius. 
As for flight software I agree.

Reply:
That's fair. The way it is right now is not very clean though (especially after reviewing some of the other code and seeing many different EARTH_RADIUS definitions). This needs to be cleaned up somehow. If you want different instances of it, perhaps you could use EARTH_RADIUS_TEST_ONLY